v2/src/components/VCard.vue

```diff
@@ -181,40 +187,35 @@ const hasDetails = computed(() => {
       </div>
     </template>
     <div v-if="hasDetails" class="card-details">
-      <div>
-        <div v-if="tags || sponsored" class="card-tags">
+      <div v-if="eyebrow" class="card-eyebrow type-body" v-html="eyebrow" />
+      <div v-if="title" class="card-title" role="heading" aria-level="3">
+        <template v-if="tags || sponsored">
           <v-tag
```

Comment:
wrap these <v-tag>s in a div with 100% width so by default, it is above the TITLE... but then you can override it for the Gothamist inline tag style.